v2/css/main.less

```diff
@@ -121,6 +121,7 @@ header[role="header"] {
 			span { color: @accent2; font-weight: 300; }
  		}
  		small { display: block; font-size: 13px; color: @bg; }
+        img {margin-top: -6px; margin-left: -20px; margin-right: 10px;}
```

Comment:
http://openresty.org/en/

the logo position is not ok.  the css file is not upgraded.